bp/tests/Resource Files/testConfig-busted.json

```diff
@@ -1,9 +1,10 @@
 {
-    "app": ".",
+    "app": "/Users/khu/Library/Developer/Xcode/DerivedData/voyager-frhgmrtfxqiflycndwcgfpqkbhdy/Build/Products/Debug-iphonesimulator/LinkedIn.app",
     "no-split": [
                  "VoyagerTests"
                  ],
     "error-retries": 0,
+    "num-sims": 2,
```

Comment:
Ditto

Reply:
Yeah, I am making sure the test doesn't fail for other reasons before it gets to validate the `runtime` config.